fhir-examples/src/main/java/com/ibm/fhir/examples/Index.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * (C) Copyright IBM Corp. 2019
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.examples;
+
+public enum Index {
+    /**
+     * Both R4 spec and IBM generated examples
+     */
+    ALL_JSON("/all-json.txt"),
+
+    /**
+     * Small mix of spec and IBM examples used for unit tests to keep build times short
+     */
+    MINIMAL_JSON("/minimal-json.txt"),
+
+    /**
+     * All R4 spec examples
+     */
+    SPEC_JSON("/spec-json.txt"),
+
+    /**
+     * All IBM generated examples
+     */
+    IBM_JSON("/ibm-json.txt"),
+
+    /**
+     * R4 spec and IBM examples less than 1MB - used for concurrency tests
+     */
+    PERFORMANCE_JSON("/performance-json.txt"),
+
+    /**
+     * Both R4 spec and IBM generated examples
+     */
+    ALL_XML("/all-xml.txt"),
+
+    /**
+     * Small mix of spec and IBM examples used for unit tests to keep build times short
+     */
+    MINIMAL_XML("/minimal-xml.txt"),
+
+    /**
+     * All R4 spec examples
+     */
+    SPEC_XML("/spec-xml.txt"),
+
+    /**
+     * All IBM generated examples
+     */
+    IBM_XML("/ibm-xml.txt");
+
+
```

Comment:
Can we merge the index files(for both json and xml) to one set instead of 2? currently, we have to double the efforts to make index changes to both.
e.g, with index content like:
       xxxxx-xxxxxx           (both json and xml)
       xxxxx-xxxxx.json    (json only) 
       xxxxxxx-xxxxxxx.xml (xml only)
       ...

Just for discussion as an potential enhancement, not a must.

Reply:
yes, this approach now makes that possible.  we can slide/dice the examples in any way we want...its as simple as just copy/pasting the lines from one to the other.  do we want to do that?

i think the format (json vs xml) doesn't really matter for any of the tests beyond the fhir-model test, and i don't want to make the MINIMAL index too large since we run that on every build.  Maybe we combine the `ibm` and `spec` index files, but leave MINIMAL and ALL separate?

Reply:
MINIMAL-json.txt + MINIMAL-xml.txt --> MINIMAL.txt
the same for other index files, because seems the file names in them are all the same, just file extension is different... 
and we just need to change the driver a little bit to process the merged index file... 
Anyway, this is just a enhancement suggestion, not really needed for this PR.

Reply:
the set isn't exactly the same.  there are some spec examples that have json and no xml and vice versa.